proteus/SpatialTools.py

```diff
@@ -119,6 +119,10 @@ def _checkFlags(self, flagSet):
         checkFlag = min(flagSet)
         assert checkFlag == 1, 'Minimum boundary/region tag/flag must be 1'
         for flag in flagSet:
+            # hack for SWFlow BCs
+            if (flag==99):
```

Comment:
I think you should open up a corresponding issue to remove this hack, because I think it leads to unnecessarily large arrays. We talked briefly about this in the meeting, but we can probably migrate to using a different data structure for handling flags so that arrays don't wind up being formed based on the maximum flag value.

Reply:
+1

Reply:
@ejtovar you could just add the issue then mark this as resolved.

Reply:
@zhang-alvin When the flag 99 is read, it only adds an extra entry to the boundary tags so that the size is size(normalTags)+1 and not size(normalTags)+99. I'll go ahead open the issue anyway so we can generalize this later.